client/starwhale/base/bundle.py

```diff
@@ -76,9 +76,10 @@ def list(
         project_uri: URI,
         page: int = DEFAULT_PAGE_IDX,
         size: int = DEFAULT_PAGE_SIZE,
+        _filter: t.Dict[str, t.Any] = {},
```

Comment:
Why use the private flag for the `_filter` argument?

Reply:
done